src/styles/global.css

```diff
@@ -14,63 +14,6 @@
     "Segoe UI Symbol", "Noto Color Emoji";
 }
 
-@keyframes marquee {
```

Comment:
To expand on Jelle's comment about atomic commits - that means fixes like these should be squashed into the original commit that introduced the mistake, i.e. the tailwind v4 commit.